exp/addons/controllers/clusterresourceset_controller_test.go

```diff
@@ -318,11 +318,18 @@ metadata:
 			}
 			return errors.Errorf("ClusterResourceSet binding does not have any resources matching %q: %v", newCMName, binding.Spec.Bindings)
 		}, timeout).Should(Succeed())
-		Expect(testEnv.Delete(ctx, testConfigmap)).To(Succeed())
+
+		By("Verifying ClusterResourceSetBinding is deleted when its cluster owner reference is deleted")
```

Comment:
Is the garbage collector running on this test suite? I didn't think it was normally being run in envtest style environments. Or is the CRS controller doing this cleanup?

Reply:
`ClusterResourceSetBinding` is deleted by CRS controller.